test/models/task_test.rb

```diff
@@ -33,4 +33,16 @@ class TaskTest < ActiveSupport::TestCase
     task.update(description: "New description")
     assert_equal "New description", task.description
   end
+
+  test "invalid without name" do
+    task = build(:task, name: nil)
+    refute task.valid?, "task is valid without a name"
+    assert_not_nil task.errors[:name], "no validation error for name present"
+  end
+
+  test "invalid without description" do
+    task = build(:task, description: nil)
+    refute task.valid?, "task is valid without description"
+    assert_not_nil task.errors[:description], "no validation error for description present"
+  end
```

Comment:
You can use https://github.com/thoughtbot/shoulda-matchers which makes this a bit faster  and less having to write our own tests.